src/main/java/org/jabref/JabRefException.java

```diff
@@ -29,11 +32,13 @@ public JabRefException(String message, String localizedMessage, Throwable cause)
     public JabRefException(Throwable cause) {
         super(cause);
     }
-
+     
+    // To display localized message on error
```

Comment:
No comment needed as already defined in `Exception` base class.

Reply:
okay.